src/main.js

```diff
@@ -22,9 +22,12 @@ const importSuite = (name, testPath, opt) => {
   describe(name, require(testPath)(opt).describe)
 }
 
-// Create dashboard application
-// importSuite('Test Resin', './resin/application.js')
-//
+describe('Preparing test environment', function () {
+  this.timeout(900000)
+
+  importSuite('Authenticate user using token', './resin/auth.js')
```

Comment:
A test suite should not be used to prepare the environment. You could use Mocha's `before` instead